source/location.html.slim

```diff
@@ -0,0 +1,18 @@
+---
+title: Location
+---
+
+section.h-lots-of-padding-top
+  .body-inner-wrapper
+    .body-inner-wrapper__section#about
+      h2 📕 About Wellington
+      p Wellington is the capital of New Zealand. It's a walkable city with a thriving Ruby community, famous for excellent coffee, howling winds, and innovative technology companies.
+      p If you're visiting from out of town:
+      ul.t-list
+        li You can get around the city on <a href="https://www.metlink.org.nz/">Metlink</a> buses. You'll want to get a <a href="https://www.snapper.co.nz/">Snapper</a> card at the airport for this.
+        li Don't bring an umbrella unless it's a <a href="https://www.bluntumbrellas.com/nz/">Blunt</a> umbrella or you don't mind it being blown inside out.
+    .body-inner-wrapper__section#signup
+      h2 ✨ About Te Papa
+      p <a href="https://www.tepapa.govt.nz/">Te Papa</a> is the national museum of Aotearoa New Zealand. It features treasures such as an <a href="https://www.tepapa.govt.nz/visit/whats-on/exhibitions/colossal-squid">enormous squid</a>. If you're visiting from out of town, we highly recommend staying for a bit longer and exploring the collection.
```

Comment:
Should we use Te Papa's full name here? Also maybe Colossal Squid rather than enormous?